test/kit/read/main.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/amp-labs/connectors"
+	"github.com/amp-labs/connectors/common"
+	ap "github.com/amp-labs/connectors/providers/kit"
```

Comment:
I see you are using `ap` as an alias. Could you tell me what does the abbreviation stand for?
However, I don't think there is a good name. I kept `testConn` in the tests, but I am not a fan ahaha.

Reply:
I initially used 'ap' as an abbreviation for 'App Provider' in my mind, but I'll change it to 'testConn' as you did in previous connectors.